components/library/zeromq/zeromq3/Makefile

```diff
@@ -34,31 +39,63 @@ COMPONENT_LICENSE_FILE= COPYING.LESSER
 COMPONENT_LICENSE= LGPLv3+
 COMPONENT_CLASSIFICATION= System/Libraries
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/configure.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+# Note: This USR path is used in build prefix and in p5m manifest,
+# except where we mediate the links and the relative SUB path is used
+COMPONENT_SUB_DIR="zeromq-$(COMPONENT_VERSION_MAJ_MIN)"
+COMPONENT_USR_DIR="$(USRDIR)/$(COMPONENT_SUB_DIR)"
+
+PKG_OPTIONS +=	-D COMPONENT_VERSION_ABI="$(COMPONENT_VERSION_ABI)"
+PKG_OPTIONS +=	-D COMPONENT_VERSION_MAJ_MIN="$(COMPONENT_VERSION_MAJ_MIN)"
+PKG_OPTIONS +=	-D COMPONENT_SUB_DIR="$(COMPONENT_SUB_DIR)"
+PKG_OPTIONS +=	-D COMPONENT_USR_DIR="$(COMPONENT_USR_DIR)"
+
+COMPONENT_PREP_ACTION =	\
+	( cd $(@D) && \
+	  libtoolize --copy --force && \
+	  aclocal -I m4 && \
+	  autoheader && \
+	  automake -c -f -a && \
+	  autoconf )
 
-PKG_OPTIONS +=	-D COMPONENT_VERSION_MAJ="$(COMPONENT_VERSION_MAJ)"
+COMPONENT_PRE_CONFIGURE_ACTION =	($(CLONEY) $(SOURCE_DIR) $(@D))
 
-COMPONENT_PRE_CONFIGURE_ACTION =	($(CLONEY) $(SOURCE_DIR) $(@D) && cd $(@D) && autoconf )
+# Build docs just once (for one ARCH), parameter varies from version to version
+CONFIGURE_OPTIONS.32 +=	--with-documentation --with-docs
+CONFIGURE_OPTIONS.64 +=	--without-documentation --without-docs
 
-CONFIGURE_OPTIONS +=	--with-documentation
```

Comment:
first CONFIGURE_OPTIONS should be = or you set prefix and other default directories twice